libgalois/include/katana/PropertyGraph.h

```diff
@@ -29,18 +29,20 @@ ProjectAsArrowArray(const T* buf, const size_t len) noexcept {
   return std::make_shared<ArrowArrayType>(len, arrow::Buffer::Wrap(buf, len));
 }
 
-/// A graph topology represents the adjacency information for a graph in CSR
-/// format.
-class KATANA_EXPORT GraphTopology {
-public:
+struct KATANA_EXPORT GraphTopologyTypes {
```

Comment:
What's the value of having this in a separate container struct?

Reply:
Other topologies inherit this to get the typedefs.